src/content/ja/2020/privacy.md

```diff
@@ -0,0 +1,298 @@
+
+WhoTracksMeのトラッカーリストには、トラッカーが所属するカテゴリーも定義されています。CDNとホスティングサイトを統計から除外すると、これらのサイトはトラッキングを行わないか、少なくともそれが主な機能ではないと仮定すると、トップ10の見方が少し変わります。
+
+{{ figure_markup(
+  image="privacy-biggest-third-party-trackers.png",
+  caption="トップ10トラッカー",
+  description="モバイルとデスクトップのクライアントで使用されているもっとも人気のある10種類のトラッカーの普及率を示す棒グラフ。デスクトップとモバイルの差はほとんどなく、モバイルではgoogle_analyticsが65.5％、doubleclickが50.0％、googleが47.6％、google_tag_managerが42.4％、facebookが30.9％、google_adservicesが19.2％、youtubeが12.7％、google_syndicationが19.2％、twitterとwordpress_statsがともに6.5％となっています。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=1831606887&format=interactive",
+  sheets_gid="1677398038"
+  )
+}}
+
+ここでは、トップ10のドメインのうち7つをGoogleが占めています。次の図は、潜在的なトラッカーの数が多い100件について、カテゴリー別の分布を示したものです。
+
+{{ figure_markup(
+  image="privacy-tracker-categories.png",
+  caption="ポテンシャルトラッカー100選のカテゴリー",
+  description="ウェブ上の潜在的なトラッカーのトップ100の分布を示す棒グラフで、広告が56、cdnが11、site_analyticsが9、ソーシャルメディアとmiscの両方が6、essentialとcustomer_helpの両方が3、オーディオとビデオの両方が2、コメントとundefinedの両方が1となっています。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=1117413918&format=interactive",
+  sheets_gid="1431872451",
+  )
+}}
+
+もっとも人気のあるトラッカーの60％近くが広告関連である。これは、オンライン広告市場の収益性がトラッキングの量に関係していると認識されているためと考えられる。
+
+### Cookies
+
+WebサイトでHTTPのレスポンスヘッダーに設定されているもっとも一般的なCookieを、その名前とドメイン別に調べました。
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>ドメイン</th>
+        <th>Cookieの名称</th>
+        <th>ウェブサイト</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>doubleclick.net</td>
+        <td><code>test_cookie</code></td>
+        <td class="numeric">24%</td>
+      </tr>
+      <tr>
+        <td>facebook.com</td>
+        <td><code>fr</code></td>
+        <td class="numeric">10%</td>
+      </tr>
+      <tr>
+        <td>youtube.com</td>
+        <td><code>VISITOR_INFO1_LIVE</code></td>
+        <td class="numeric">10%</td>
+      </tr>
+      <tr>
+        <td>youtube.com</td>
+        <td><code>YSC</code></td>
+        <td class="numeric">10%</td>
+      </tr>
+      <tr>
+        <td>doubleclick.net</td>
+        <td><code>IDE</code></td>
+        <td class="numeric">9%</td>
+      </tr>
+      <tr>
+        <td>doubleclick.net</td>
+        <td>unknown</td>
+        <td class="numeric">9%</td>
+      </tr>
+      <tr>
+        <td>youtube.com</td>
+        <td><code>GPS</code></td>
+        <td class="numeric">9%</td>
+      </tr>
+      <tr>
+        <td>doubleclick.net</td>
+        <td>unknown</td>
+        <td class="numeric">8%</td>
+      </tr>
+      <tr>
+        <td>google.com</td>
+        <td><code>NID</code></td>
+        <td class="numeric">6%</td>
+      </tr>
+      <tr>
+        <td>doubleclick.net</td>
+        <td>unknown</td>
```

Comment:
Can we translate all instanced of "unknown"?

Reply:
@bazzadp Yes, We Japanese understand "unknown".

Reply:
OK so just leave it as is and merge?

Reply:
No problem. 👍

Reply:
Sorry bit confused. So that is a "Yes" to merging it as is?

Reply:
Yes, please merge.